packages/ERTP/src/paymentLedger.js

```diff
@@ -15,13 +15,15 @@ import { BrandI, makeIssuerInterfaces } from './typeGuards.js';
 /**
  * @import {Amount, AssetKind, DisplayInfo, PaymentLedger, Payment, Brand, RecoverySetsOption, Purse, Issuer, Mint} from './types.js'
  * @import {ShutdownWithFailure} from '@agoric/swingset-vat'
- * @import {Key} from '@endo/patterns';
+ * @import {TypedPattern} from '@agoric/internal';
  */
 
 /**
+ * @template {AssetKind} K
```

Comment:
Generally, which I've written code like this template declaration, I revise it to

```suggestion
 * @template {AssetKind} [K=AssetKind]
```

since the supertype constraint is usually also a fine default.

Reply:
In this case there's no way the to omit the parameter because it's positional